### DISCUSSION TITLE
Add from URL?

### DISCUSSION THREAD

**Discussion by timboswell** [mediacms-io/mediacms#428] [Feature Requests]
I'm considering migrating over from PeerTube to MediaCMS. I love the interface and the ability to upload multiple formats, but the one thing that's missing for me is the ability to easily add existing Youtube videos without ripping them first. 

Are there any plans to incorporate Youtube-dl and allow adding media directly from a URL?

**Comment by mgogoulos** [comment]
Hi, this is a good feature that is missing indeed. Could be implemented the other way around too (publish to Youtube). I am available to provide mentoring if anyone is willing to implement it!
